src/LiveComponent/src/Resources/views/form_theme.html.twig

```diff
@@ -1,19 +1,13 @@
 {%- block live_collection_widget -%}
     {{ block('form_widget') }}
-    {%- if button_add_prototype is defined and not button_add_prototype.rendered -%}
-        {{ form_row(button_add_prototype, { attr: button_add_prototype.vars.attr|merge({
-            'data-action': 'live#action',
-            'data-action-name': 'addCollectionItem(name=' ~ form.vars.full_name ~ ')'
-        }) }) }}
+    {%- if skip_add_button|default(false) is same as false and button_add_prototype is defined and not button_add_prototype.rendered -%}
+        {{ form_row(button_add_prototype) }}
```

Comment:
What do you think about changing the name of the exposed variable to `button_add` or maybe something more specific to the collection `live_collection_item_add` or just `item_add`?

`skip_add_button`, `button_add`, `button_delete`?

Reply:
`add_button`, `delete_button`?

Reply:
Actually I kind of like `button_add` and `button_delete` better, but not sure about it :smiley:

Reply:
I'm fine with that.